monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md

```diff
@@ -57,47 +53,47 @@ Monero 一詞是取自世界語(Esperanto)。之所以選擇使用世界語是
 
 ### MONERO 的關鍵特別之處
 
-**Monero 使用CryptoNote’代码基库：**這和比特幣或乙太幣或許多從他們衍生出來的貨幣相比，有著截然不同的程式碼架構，這是為了做到更好的保護隱私功能。
+**Monero 使用 CryptoNote 程式碼：** 這和比特幣或乙太幣或許多從他們衍生出來的貨幣相比，有著截然不同的程式碼架構，這是為了做到更好的保護隱私功能。
 
-**隱私是預設必要的；而透明性是可以自行選擇的。Monero 的設計可以混淆交易資訊的每一個層面：**付款者、收款者、還有交易本身。如果使用者需要透明性，他們可以選擇公開他們的唯讀錢包以揭露該錢包的收款資訊。
+**隱私是預設必要的；而透明性是可以自行選擇的：** Monero 的設計在每一個層面都是保護隱私的：付款者、收款者、還有交易本身。使用者可以選擇建立並公開他們的唯讀錢包(view-only wallet)以揭露該錢包的收款資訊或利用查看金鑰(view-keys)揭露特定的交易。
 
-**例行性網路協定升級：**Monero的社群開發者定期地進行網路協定升級(硬分叉)以確保所有使用者都能享受到最佳的安全、隱私保護及其他功能。這樣做法讓 Monero 能夠更安全也更靈活地去應對任何可能興起的安全性威脅。**[那些我曾聽說的許多分叉幣是什麼？](https://bitcoinmagazine.com/articles/monero-just-hard-forked-and-it-resulted-four-new-projects/)**
+**例行性網路協定升級：** Monero的社群開發者定期地進行網路協定升級(硬分叉)以確保所有使用者都能享受到最佳的安全、隱私保護及其他功能。這樣做法讓 Monero 能夠更安全也更靈活地去應對任何可能興起的安全性威脅。 **[那些我曾聽說的許多分叉幣是什麼？](https://bitcoinmagazine.com/articles/monero-just-hard-forked-and-it-resulted-four-new-projects/)**
 
-**Monero 區塊獎勵策略：**區塊獎勵會緩慢地下降直到2022年的五月底會固定在尾發行的速率，約為 0.6 XMR / 每區塊。尾發行將會持續提供永久的挖礦動機。此外，更重要的是，尾發行也提供了 Monero 一個內建穩定且可預測的通膨，這是一個真正的貨幣所需要的特性。
+**Monero 區塊獎勵策略：** 區塊獎勵會緩慢地下降直到2022年的五月底會固定在尾發行的速率，約為 0.6 XMR / 每區塊。尾發行將會持續提供永久的挖礦動機。此外，更重要的是，尾發行也提供了 Monero 一個內建穩定且可預測的通膨，這是一個真正的貨幣所需要的特性。
 
-**Monero 研究室：**Monero 不僅僅是要打造一款有替代性的貨幣，更是要持續探索加密貨幣的金融隱私領域。為了達到這點， Monero 社群中的一些數學與計算物理學家們發表了**[白皮書](https://lab.getmonero.org/)**，揭露數個正在進行中的**[研究目標](https://lab.getmonero.org/)**。
+**Monero 研究室：** Monero 不僅僅是要打造一款有替代性的貨幣，更是要持續探索加密貨幣的金融隱私領域。為了達到這點， Monero 社群中一些在數學、物理、資安和區塊鏈計算領域專業的學術研究者們發表了 **[研究論文](https://lab.getmonero.org/)** ，揭露數個正在進行中的 **[研究目標](https://lab.getmonero.org/)** 。
 
-**低門檻的挖礦：**人人只要有一台連上網路的電腦或是網頁瀏覽器就能參與挖礦。
+**低門檻的挖礦：** 人人只要有一台連上網路的電腦或是網頁瀏覽器就能參與挖礦。
 
 ### 真實世界中的應用
 
-由於 Monero 是安全、低手續費、無疆界的貨幣，人們可以自由發送交易，不受到貪腐、破產的政府或銀行的影響。這提供了身處於**[苛政的國家](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)**或經濟蕭條的人民一個獨立的金融管道。
+由於 Monero 是安全、低手續費、無疆界的貨幣，人們可以自由發送交易，不受到貪腐、破產的政府或銀行的影響。商業活動也不會被競爭對手所窺探金融機密。這提供了身處於 **[苛政的國家](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)** 、經濟蕭條或商業競爭激烈環境的人民和商業組織一個獨立的金融管道。
 
-完全隱匿保護的金融歷史記錄保護消費者與公司不受到價格操控、原物料鏈暴露、經濟歧視等等的傷害。**[Monero是目前唯一在加密貨幣中](https://www.reddit.com/r/Monero/comments/8k8pk9/monero_the_worlds_bestkept_secret/)**可以提供完全的可替代性、去中心化的電子現金。
+完全隱匿保護的金融歷史記錄保護消費者與公司不受到價格操控、原物料鏈暴露、經濟歧視等等的傷害。 **[Monero是目前唯一在加密貨幣中](https://www.reddit.com/r/Monero/comments/8k8pk9/monero_the_worlds_bestkept_secret/)** 可以提供完全的可替代性、去中心化的電子現金。
 
 ### 技術性基本資料
 
-_(資料截至 11/14/2018)_
+_(資料截至 2/1/2019)_
 
-+ **活耀節點數量：** 1,756 (來源:https://monerohash.com/nodes-distribution.html)
++ **活耀節點數量：** 1,564 (來源:https://monerohash.com/nodes-distribution.html)
 
-+ **交易網路算力：** 435.17 MH/s
++ **交易網路算力：** 663.2 MH/s
 
-+ **平均每小時交易數：** 178 (天平均)
++ **平均每小時交易數：** 149 (天平均)
 
-+ **全網挖礦CPU核心數：** 14,499,178
++ **全網挖礦CPU核心數：** 22,234,297
 
-+ **貨幣流通量：** 16,570,126 XMR (估計值)
++ **貨幣流通量：** 16,758.217 XMR (估計值)
 
-+ **貨幣市值：** $1,503,928,296 USD (約佔0.77%加密貨幣總市值)
++ **貨幣市值：** $743,944,945 USD (約佔0.77%加密貨幣總市值)
```

Comment:
We found the information we had posted was incorrect and thanks to @el00ruobuob we were able to fix it. This is the string with the correct information.

Market Capitalization: $732,932,071 USD (~0.65% of total cryptocurrency market cap)

Reply:
Revised.